src/VisualStudio/Core/Def/Implementation/Progression/GraphQueries/InheritedByGraphQuery.cs

```diff
@@ -41,7 +36,7 @@ internal sealed class InheritedByGraphQuery : IGraphQuery
                 else if (namedType.TypeKind == TypeKind.Interface)
                 {
                     var derivedTypes = await namedType.FindDerivedInterfacesAsync(solution, cancellationToken).ConfigureAwait(false);
```

Comment:
Unrelated to your change, but in Dev12 the Derived Types query we used to include classes that implemented the interface, but in the same hierarchical fashion that you're doing here now (as opposed to the new Implemented By query where flatten the list of implementers). Let's consider bringing that back.